sdk/keyvault/azure-security-keyvault-keys/src/cryptography/rsa_cryptography_provider.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "azure/keyvault/keys/cryptography/rsa_cryptography_provider.hpp"
+
+#include <memory>
+#include <string>
+
+namespace Azure {
+  namespace Security {
+    namespace KeyVault {
+      namespace Keys {
+        namespace Cryptography {
+  namespace _detail {
+
+    EncryptResult RsaCryptographyProvider::Encrypt(
```

Comment:
Where do you initialize the `iv` for AES-CBC(PAD)? I don't see that anywhere? I reviewed all .hpp files, but looked at various .cpp files where I thought this might be, but didn't see it.

Reply:
haven't tested AES :)  Will do it soon and will fix the iv init and the other comments.  Thank you!